### PR TITLE
change announce for obstacle stop

### PR DESCRIPTION
**Pull request by JunichiMorimoto**
## PR Type

- Improvement

## Related Links
https://tier4.atlassian.net/browse/RT4-7570

## Description

- use temporary_stop.wav for obstacle stop
- change function name: not_timeout to in_interval
- use constant for MotionState

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py [hunk 1]
```diff
@@ -95,7 +96,7 @@ def reset_all_timeout(self):
             duration = getattr(self._mute_parameter, attr)
             setattr(self._timeout, attr, self._node.get_clock().now() - Duration(seconds=duration))
 
-    def not_timeout(self, timeout_attr):
+    def in_interval(self, timeout_attr):
```
Comment (JunichiMorimoto) [inline src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py:99]: not_timeoutがしっくり来なかったので in_intervalに変更しています（grep確認済み）

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def __init__(
             node.get_clock().now(),
         )
         self._in_emergency_state = False
-        self._prev_motion_state = 0
+        self._prev_motion_state = MotionState.UNKNOWN
```
Comment (JunichiMorimoto) [inline src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py:66]: motion_stateをハードコードしていた部分を MotionStateから取るようにしています

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py [hunk 3]
```diff
@@ -336,21 +338,20 @@ def stop_reason_checker_callback(self):
...
+                    "Intersection",
+                    "MergeFromPrivateRoad",
+                    "Crosswalk",
+                    "Walkway",
+                    "StopLine",
+                    "NoStoppingArea",
+                    "TrafficLight",
+                    "BlindSpot",
                 ]
                 and self._autoware.information.velocity == 0
```
Comment (yabuta) [inline src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py:350]: 個々の停止した判定をmotion state==STOPにしたほうがいいかと思います

**Review by yabuta — CHANGES_REQUESTED** [review]

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py (hunk 3 above)
Comment (JunichiMorimoto) [reply src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py]: 対応しました！

File: src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py [hunk 4]
```diff
@@ -336,21 +338,20 @@ def stop_reason_checker_callback(self):
...
                     "BlindSpot",
                     "BlockedByObstacles",
+                    "Intersection",
+                    "MergeFromPrivateRoad",
+                    "Crosswalk",
+                    "Walkway",
+                    "StopLine",
+                    "NoStoppingArea",
+                    "TrafficLight",
+                    "BlindSpot",
```
Comment (yabuta) [inline src/vehicle_voice_alert_system/src/vehicle_voice_alert_system/announce_controller.py:348]: 多分個々の理由も発話する理由を見直してもらうと思うのですがそれはもう一つのタスクの方でやりたいと思います
Avoidance増やしてほしいとか

**Review by yabuta — APPROVED** [review]
LGTM

**Review by tkhmy — APPROVED** [review]
LGTM

**Pull request merged**